### ISSUE TITLE
redacted/waffles ready

### ISSUE THREAD

**Issue by MerlijnWajer** [whipper-team/whipper#288]
I am trying to make a list of tickets we should finish before we can work with redacted/waffles on the logger part and then have whipper approved by both sites.

- [x] https://github.com/JoeLametta/whipper/issues/73 / clear tutorials (more up to the websites perhaps)
- [x] https://github.com/JoeLametta/whipper/issues/117
- [x] https://github.com/JoeLametta/whipper/issues/135
- [x] https://github.com/JoeLametta/whipper/issues/221
- [ ] https://github.com/JoeLametta/whipper/issues/174
- [ ] https://github.com/whipper-team/whipper/issues/234 (even though it's an upstream bug I think it needs to be fixed too because it affects a big number of users)
- [ ] https://github.com/JoeLametta/whipper/issues/294

Anything else? We might have to rework the logger interface so that it can also call (external) binaries.

I didn't make a Milestone for this because (AFAIK) tickets can be only part of one milestone.

**Comment by peaveyman** [comment]
We need to find out for sure which logger is going to be acceptable for redacted. And if redacted is good with it waffles will be.

**Comment by MerlijnWajer** [comment]
It seems like the current logger is fine - so we don't need #117. It's unclear if we will need #135. #221 would be nice to have, but for testing purposes we don't need it. #294 is important, though. #174 seems not that important for red right now since it mostly affects CDs with extra data, but we should get that fixed soon as well.

**Comment by peaveyman** [comment]
I can't see  how 135 and 221 would be needed.

**Comment by JoeLametta** [comment]
1. #117 I don't consider it a blocker: I'll surely update the README to clarify logger plugins current status. I may also update the [morituri-eaclogger](https://github.com/JoeLametta/morituri-eaclogger) (or discontinue it).
2. #221 I would say it's 99% done: again, not a blocker.
3. #174 Would be nice having this fixed
4. #135 Would be nice having this fixed: not a blocker.
5. #294 I think this is needed.

> Clear tutorials (more up to the websites perhaps)

Maybe #73?

**Comment by itismadness** [comment]
So in looking to implement this within a logchecker for Orpheus, one of the biggest hurdles I think we face is just knowing what are the possible logs that whipper may output that a user may check against. This includes a perfect log, but also what a log of a rip that has totally failed (or failed as much as whipper would allow while still producing a log). I can get an idea of it looking through the `logger.py` module that produces the YAML, but just concrete examples would be wonderfully helpful.

**Comment by MerlijnWajer** [comment]
Alright. Going to try to get this rolling ASAP now. Remaining three tickets are all assigned to me it seems. ;)

**Comment by nafanz** [comment]
It is wonderful that it was approved at Orpheus.
I hope that it will also be available for RED and Waffles.

**Comment by peaveyman** [comment]
I can't speak for RED but as for Waffles, when it gets back online you can expect Whipper to be an approved ripper.

**Comment by JoeLametta** [comment]
Even if it's an upstream bug I think #234 needs to be fixed too because it affects a big number of users.

**Comment by 7596ff** [comment]
Is this still a priority?

**Comment by JoeLametta** [comment]
> Is this still a priority?

Those are just some of the issues scheduled for [milestone 1.0](https://github.com/whipper-team/whipper/milestone/1). Unfortunately I don't have enough time / experience to handle everything on my own (that's why progress is slow) but I try to review every issue / pull request in a timely manner.
Of course every contribution is encouraged and appreciated.

**Comment by xavery** [comment]
> I can't speak for RED but as for Waffles, when it gets back online you can expect Whipper to be an approved ripper.

It never did. 😢

**Comment by user-1-2-3-4-5** [comment]
What is the current state of the program on RED?
